### PR TITLE
Update Grafana dashboard in dedicate folder

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description

For keeping dashboard organized, update Grafana dashboard into dedicate folder instead of General folder

### CODE REVIEW

File: .github/workflows/Nightly_Perf_Env_CI.yml [hunk 1]
```diff
@@ -391,7 +392,7 @@ jobs:
         run: |
           git diff --quiet "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/jsonnet/main.libsonnet || echo "changes=true" >> "$GITHUB_OUTPUT"
 
-      - name: Commit main.libsonnet if any changes are detected
+      - name: Commit main.libsonnet if any products versions are detected
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_CI.yml:395]: "changes to product versions"

File: .github/workflows/Perf_Env_Build_Test_CI.yml [hunk 2]
```diff
@@ -446,7 +447,7 @@ jobs:
         run: |
           git diff --quiet "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/jsonnet/main.libsonnet || echo "changes=true" >> "$GITHUB_OUTPUT"
 
-      - name: Commit main.libsonnet if any changes are detected
+      - name: Commit main.libsonnet if any products versions are detected
```
Comment (RobertKrawitz) [inline .github/workflows/Perf_Env_Build_Test_CI.yml:450]: "changes to product versions"

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
Trivial changes

File: benchmark_runner/common/grafana/grafana_operations.py [hunk 3]
```diff
@@ -79,6 +80,32 @@ def get_latest_dashboard_version(self):
         except requests.exceptions.RequestException as e:
             raise Exception(f"Error fetching dashboard version: {e}")
 
+    def get_dashboard_folder_id(self):
+        """
+        This method get dashboard folder id
```
[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/common/grafana/grafana_operations.py]: Update more clear document

**Review by RobertKrawitz — APPROVED** [review]

**Pull request merged**